pydantic/mypy.py

```diff
@@ -108,13 +108,16 @@ class PydanticPluginConfig:
     warn_untyped_fields: bool
 
     def __init__(self, options: Options) -> None:
-        if options.config_file is None:  # pragma: no cover
-            return
-        plugin_config = ConfigParser()
-        plugin_config.read(options.config_file)
-        for key in self.__slots__:
-            setting = plugin_config.getboolean(CONFIGFILE_KEY, key, fallback=False)
-            setattr(self, key, setting)
+        # NOTE: pyproject.toml reading not supported
+        if options.config_file is None or options.config_file.endswith('.toml'):  # pragma: no cover
+            for key in self.__slots__:
+                setattr(self, key, True)  # default to strict without a config
```

Comment:
Thank you for the PR!
I just reviewed the other one that adds support. I know toml is messed up but it's still stdlib library so no extra dependency on pydantic side. Once they switch to tomli, it will be in user en.
So for this PR there is still the question of "when there is no config file found" and I would have said "everything to False".
I don't even know if that's really important but you probably took more time than me to think about it. I would love to have your opinion

Reply:
>  I know toml is messed up but it's still stdlib library so no extra dependency on pydantic side

What do you mean by this? That the `toml` PyPI package is part of the stdlib? It's third party as well (https://pypi.org/project/toml/).

Just wanted to make sure there was no confusion.

Reply:
There was oops 🙈 2am...50 reviews in 1 day.
I'm going to bed!
Thanks!